tests/devices/test_default_qutrit.py

```diff
@@ -260,6 +260,277 @@ def test_apply_rotations_one_wire(self, qutrit_device_1_wire, tol):
     # TODO: Add tests for state preperation ops after they're implemented
 
 
+class TestExpval:
+    """Tests that expectation values are properly calculated or that the proper errors are raised."""
+
+    # TODO: Add test for expval of non-parametrized observables
+
+    @pytest.mark.parametrize(
+        "operation,input,expected_output,par",
```

Comment:
This is picky but the naming of parameters here is not super clear. Recommend to re-express `operation` as `observable`, `input` as something like `state`, and `par` as `matrix`, for example.

Reply:
I'm following the same naming convention here as in `test_default_qubit.py`. Perhaps I can change it if we have a response from the Xanadu team regarding updates to testing conventions (?)